io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -741,3 +774,7 @@ impl PtplFileOps for LvolPtpl {
             .with_extension("json")
     }
 }
+
+pub fn get_logical_volume(lvol: &Lvol) -> Box<&dyn LogicalVolume> {
```

Comment:
Boxing references looks a bit strange. Usually Box owns the value rather than a reference.